Dockerfile

```diff
@@ -1,6 +1,23 @@
+FROM golang:1.17 AS builder
+
+ARG VERSION=SNAPSHOT
+ENV GO111MODULE=on
+
+WORKDIR /workspace
+
+COPY go.mod go.sum /workspace/
+
+RUN go mod download
+
+COPY *.go /workspace/
+COPY cmd /workspace/cmd
+COPY plugins /workspace/plugins
+
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags="-X 'github.com/movio/bramble.Version=$VERSION'" -o bramble ./cmd/bramble
```

Comment:
Perhaps don't set the `GOARCH` as this might cause trouble with arm macs etc.

Reply:
As far as I know the image would be `amd64` regardless and arm macs can run `amd64` images. I don't have one though and it doesn't need to be there, so dropped.